code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -51,9 +51,9 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		if(reac_volume >= 5)
```

Comment:
Perhaps change this to `if((reac_volume * boozepwr) > 325)`  That way weaker booze is less potent, because right now, beer and pure ethanol are equally effective.

Reply:
325 seems like an arbitrary number. I'm sure it isn't, but what's your logic?

Reply:
Ethanol's boozepower is 65,  65*5 is 325, which would keep the same functionality as before for pure ethanol.